py/selenium/webdriver/remote/file_detector.py

```diff
@@ -17,7 +17,9 @@
 
 from abc import ABCMeta, abstractmethod
 import os
+from typing import Optional
 from selenium.webdriver.common.utils import keys_to_typing
+from selenium.types import AnyKey
```

Comment:
just realised... This breaks things since there is not selenium types module, is there a PR for this?

Reply:
Oh, sorry @AutomatedTester , my bad. Let me take a look at this immediately - will add a PR this evening. Looks like I've forgotten to include a module in the previous commits! Sorry again.

Reply:
@AutomatedTester I have resolved this in #9647. Sorry again!